configs/projects/pe-razor-server.rb

```diff
@@ -0,0 +1,19 @@
+project 'pe-razor-server' do |proj|
+  platform = proj.get_platform
+
+  proj.conflicts "razor-server"
+  proj.replaces "razor-server"
+  proj.provides "razor-server"
+  proj.conflicts "pe-razor-libs"
+  proj.replaces "pe-razor-libs"
+  proj.provides "pe-razor-libs"
+
+  proj.setting(:pe_package, true)
+  proj.setting(:razor_user, 'pe-razor')
+  proj.setting(:pe_version, ENV['PE_VER'] || '2018.1')
+  platform.add_build_repository "http://enterprise.delivery.puppetlabs.net/#{proj.pe_version}/repos/#{platform.name}/#{platform.name}.repo"
+
+  proj.instance_eval File.read('configs/projects/razor-server-shared.rb')
+
+  proj.setting(:configdir, '/opt/puppetlabs/server/apps/razor-server')
```

Comment:
This looks like it's the same as `install_root` in the shared file below, do we need a separate variable? If we do can we set `config_dir` to `install_root` so we don't have the same path hardcoded everywhere?

Reply:
I think we do need a separate variable, since configdir is different in foss and PE. but I can set this to the install root.